### ISSUE TITLE
Version command is always returning dev-main

### ISSUE THREAD

**Issue by olivernybroe** [rectorphp/rector#6434]
# Bug Report

| Subject        | Details                                                         |
| :------------- | :---------------------------------------------------------------|
| Rector version | 0.11.5             |

When running version command `dev-main` is always returned.
```
╰─ vendor/bin/rector --version
Rector dev-main

```

**Comment by TomasVotruba** [comment]
Good catch!

I've been working on the fix since yestereday. dev-main works :+1:  https://github.com/rectorphp/rector-
src/commit/e2bd8de3480bc10b518a1713987cbbbf24b54a6b#commitcomment-51008091

**Comment by olivernybroe** [comment]
Nice! 👍

**Comment by samsonasik** [comment]
It seems fixed in latest rector 0.11.7: 

```bash
$ vendor/bin/rector --version
Rector 4c5f678ca4c1d42846a80d8677718d3ac88026c9
```

I am closing it.
